sandbox/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.12
+
+# Install system dependecies
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        make
+
+ARG DIR=$(pwd)
+ADD . /firectl
+WORKDIR /firectl
+CMD TRIMPATH="$DIR" make release && cp firectl /release
```

Comment:
I think this would be better: `CMD TRIMPATH=$(pwd) make release && cp firectl /release`